### ISSUE TITLE
Publish to PECL

### ISSUE THREAD

**Issue by glensc** [NoiseByNorthwest/php-spx#134]
PECL is standard way of installing extensions for PHP.

```
pecl install spx
```

**Comment by NoiseByNorthwest** [comment]
Yes, but I've first to spend some time to meet some requirements:
https://git.php.net/?p=php-src.git;a=blob_plain;f=CODING_STANDARDS.md;hb=HEAD
https://wiki.php.net/internals/review_comments

That was the point here #26 

And then I've to start the pecl proposal process.

**Comment by glensc** [comment]
oh, what? that's super stupid!

I thought pecl point was to have a community place to distribute extensions that are not in php-src, why would pecl acceptance require coding standard of php-src.

**Comment by ruudk** [comment]
Or, just create a `package.xml` file on the root and instruct people to install it with [Pickle](https://github.com/FriendsOfPHP/pickle#introduction).

It can install from PECL but also from source:
```bash
$ pickle install https://github.com/NoiseByNorthwest/php-spx/archive/refs/tags/v0.4.10.tar.gz
  - Installing v0.4.10.tar.gz (unknown): Downloading (100%)

In PackageXml.php line 68:

  The path '/var/folders/m4/2_8czq_s1dbcfkm1yf8zjgg80000gn/T/v0.4.10.tar.gz' doesn't contain package.xml

```
